### PR TITLE
[T] (quests) Resource_gen

### PR DESCRIPTION
**Pull request by Ricky-fight**
Co-authored-by: xin_yi_hl <1012737146@qq.com> 
Co-authored-by: 藏青 <2291848664@qq.com>
Co-authored-by: Duyan <Duyan013m@gmail.com>
Co-authored-by: Cactusstudent <97865546+Cactusstudent@users.noreply.github.com>
Co-authored-by: KlparetlR <3421037143@qq.com>
Co-authored-by: MeMoXrisE qtrewq01@163.com
Co-authored-by: 琳琳 <604896778@qq.com>

### CODE REVIEW

**Comment by Ricky-fight** [comment]
看一下还有没有问题。

File: config/ftbquests/quests/chapters/resource_gen.snbt [hunk 1]
```diff
@@ -1063,26 +1063,7 @@
...
-									Count: 1b
-								}
-								{
-									id: "minecraft:prismarine_shard"
-									Count: 1b
-								}
-							]
-						}
-					}
+					item: "reliquary:guardian_spike"
```
Comment (SeeleVol) [inline config/ftbquests/quests/chapters/resource_gen.snbt:1066]: 1066-1085

File: config/ftbquests/quests/chapters/resource_gen.snbt [hunk 2]
```diff
@@ -1341,56 +1322,56 @@
...
-					Count: 1b
-					tag: { }
-				}
-			}]
-			rewards: [{
-				id: "5199A98CF47A83D9"
-				type: "item"
-				item: {
+				}		
+		]
```
Comment (SeeleVol) [inline config/ftbquests/quests/chapters/resource_gen.snbt:1326]:
```suggestion
	1326-1356少一个缩进	]
```

File: config/ftbquests/quests/chapters/resource_gen.snbt (hunk 1 above)
Comment (xinyihl) [inline config/ftbquests/quests/chapters/resource_gen.snbt:1066]:
```suggestion
										item: {
						id: "itemfilters:or"
						Count: 1b
						tag: {
							items: [
								{
									id: "reliquary:guardian_spike"
									Count: 1b
								}
								{
									id: "minecraft:prismarine_crystals"
									Count: 1b
								}
								{
									id: "minecraft:prismarine_shard"
									Count: 1b
								}
							]
						}
					}
```
这些这么没了？？？

File: config/ftbquests/quests/chapters/resource_gen.snbt [hunk 3]
```diff
@@ -111,15 +111,15 @@
...
-			title: "Easy Tree Automation!"
+			title: "简单树场！"
 			x: -5.5d
 			y: 0.0d
 			shape: "square"
-			subtitle: "Make a Bonsai Pot"
+			subtitle: "做一个盆栽盆"
 			description: [
-				"The &bBonsai Pot&r grows trees (and tree-adjacent plants) automatically, using soil and a sapling."
+				"这个&b盆栽盆&r用土壤和一个树苗自动地种植树（或树苗类似地植物）。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/resource_gen.snbt:120]:
```suggestion
				"这个&b盆栽盆&r用土壤和一个树苗自动地种植树（或与树苗类似的植物）。"
```

**Pull request merged**